.github/codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
```

Comment:
you should update the `codecov.yml` in working dir, rather than `.github/codecov.yml`

Reply:
Let me take a shot.